controllers/templatesync/template_sync.go

```diff
@@ -176,7 +225,7 @@ func (r *PolicyReconciler) Reconcile(ctx context.Context, request reconcile.Requ
 		}
 
 		existingDep, ok := topLevelDeps[depID]
-		if ok && existingDep != dep.Compliance {
+		if ok && existingDep != string(dep.Compliance) {
```

Comment:
What prompted this change since it's already a string?

Also, @JustinKuli as a side note, should this also be changing these to lowercase for comparison because I don't see any restrictions of the user specifying `complaint` vs `Compliant`.

Reply:
The `Compliance` in the new propagator was updated to an `enum`, so it has to be converted to a string to compare it.

Reply:
(I bumped the Propagator version to pick up updated test utils.)

Reply:
The enum applies to dependencies, so the case checking shouldn't be required: https://github.com/open-cluster-management-io/governance-policy-propagator/pull/80/files

Reply:
Alternatively, should `existingDep` be converted to `policiesv1.ComplianceState` instead?